packages/cli/src/build-single-file.ts

```diff
@@ -213,6 +213,12 @@ export function buildSingleFile({
             projectAssets.add(resolve(fileDirectory, url));
         }
     }
+    // add native css imports as assets
+    for (const { request } of res.meta.getImportStatements()) {
+        if (request.endsWith('.css') && !request.endsWith('.st.css') && isAsset(request)) {
```

Comment:
It's problematic to use the request to determine if a file is Stylable. we should first resolve the request and check on that